addons/csi/vsphere/csiController.yaml

```diff
@@ -28,6 +25,9 @@
 {{ if eq .Cluster.MajorMinorVersion "1.21" }}
 {{ $version = "v2.2.1 "}}
 {{ end }}
+{{ if eq .Cluster.MajorMinorVersion "1.22" }}
+{{ $version = "v2.2.1 "}}
+{{ end }}
```

Comment:
This is a bit more complicated because vSphere CSI doesn't support 1.22. We should block upgrades to 1.22 for clusters using vSphere CSI. This has been discussed internally, but we need to follow up on it.